### DISCUSSION TITLE
cache sugestion

### DISCUSSION THREAD

**Discussion by mrcode0x0** [Nicasiomarques/github-user-search-app#14] [Q&A] [answered]
Hello, I loved the project, I had a doubt about the cache part because it was not implemented using localstorage since the map only keeps the data in memory because after the browser updates the data will be lost and that can be a problem I think.

Once again your project is incredible keep up the great work.

**Comment by Nicasiomarques** [comment, marked as answer]
Thank you very much, we will take it into account, could you please open an issue explaining more details.
